src/fireblocks-sdk.ts

```diff
@@ -1159,6 +1161,13 @@ export class FireblocksSDK {
         return await this.apiClient.issueGetRequest(url);
     }
 
+    /**
+     * Get addresses for a vault account for asset with paging
```

Comment:
let's use something similar to what u used in dev-api \ yaml (for the Doc) : 

"Gets a paginated response of the addresses for a given vault account and asset"